lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1034,8 +1034,8 @@ public function getDateDiffExpression($date1, $date2)
     /**
      * Returns the SQL to add the number of given seconds to a date.
      *
-     * @param string  $date
-     * @param integer $seconds
+     * @param string         $date
+     * @param integer|string $seconds
```

Comment:
As far as I know, there is no Expr to represent datetime arithmetics. Field reference can not be represented as a data type either. Leaving this just an integer does not reflect how this method can be used. I can not see any good way of doing this PHPDoc at all apart of introducing datetime Expr that goes far beyond this issue. What would you suggest?